app/src/androidTest/java/org/mozilla/tv/firefox/ui/robots/NavigationOverlayRobot.kt

```diff
@@ -97,6 +102,23 @@ class NavigationOverlayRobot {
     fun disableSessionIdling(activityTestRule: MainActivityTestRule) { activityTestRule.loadingIdlingResource.ignoreLoading = true }
     fun enableSessionIdling(activityTestRule: MainActivityTestRule) { activityTestRule.loadingIdlingResource.ignoreLoading = false }
 
+    // Focus the pin button using keypresses, starting from urlbar assuming back is disabled, forward is enabled
+    fun navigateToPinButton() {
+        device.apply {
+            pressDPadUp()
+            repeat(2) { pressDPadRight() }
```

Comment:
hmm, isn't this same as remoteRight()?  wouldn't it be better to change to remoteRight(int x) so it can repeat right button tap x times?  (Or as I said above, use pressDPadUp() and remoteRight(2) in the main test script with comments